ulii/views/redirect.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import get_object_or_404
+from django.views.generic import RedirectView
+
+from peachjam.models import Judgment
+
+COURT_CODE_MAPPINGS = {}
+
+
+class RedirectCaseURLsView(RedirectView):
+    pattern_name = "document_detail"
+
+    def get_redirect_url(self, court, year, number, *args, **kwargs):
+        court_code = COURT_CODE_MAPPINGS[court]
+        expression_frbr_uri = (
+            f"/akn/ug/judgment/{court_code}/{year}/{number}/eng@{year}"
```

Comment:
@longhotsummer We need to have the full date for this expression_frbr_uri to work. We only have year here.

Reply:
We are also assuming the language

Reply:
I have changed the code to search using the year and and frbr_uri_number.

Reply:
Peachjam accepts a work FRBR URI and will find the latest date and best language to use. So `/akn/ug/judgment/abc/2022/3` will redirect as appropriate.

eg. visit https://lawlibrary.org.za/akn/za/act/1996/constitution and you'll see it redirect.

Reply:
Awesome. I didn't know we did this. :+1:

Reply:
Changed :heavy_check_mark: